src/ashwat.cpp

```diff
@@ -231,7 +231,7 @@ int Solve(			// matrix solver
 		SwapRows( L, NOS);
 		for (I=LP; I<=N; I++)
 		{	double Y = M( I, L) / M( L,L);
-			for (int J=L; J<=NP2; J++)
+			for (J=L; J<=NP2; J++)
```

Comment:
Instead of using the larger scope, make all the loop scopes smaller: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es5-keep-scopes-small